Source/Human/Human.cpp

```diff
@@ -6,15 +6,24 @@
 */
 
 #include "Human.h"
-
+#include <stdlib.h>
+#include <time.h>
 
 Human::Human(): Actor()
 {
+  srand(time(NULL));
```

Comment:
@arive298 I have created an interface for the random numbers, and I think we can use it here.